Examples/Tests/particle_boundary_interaction/analysis.py

```diff
@@ -29,20 +29,20 @@
 z_analytic = -0.20531
 
 print("NUMERICAL coordinates of the point of contact:")
-print("x=%5.5f, y=%5.5f, z=%5.5f" % (x[0], y[0], z[0]))
+print(f"x={x:5.5f}, y={y:5.5f}, z={z:5.5f}")
```

Comment:
Was the `[0]` index dropped intentionally here?

Reply:
Fixed in c1266d15990fa.